pkgs/tools/virtualization/google-compute-guest-configs/default.nix

```
@@ -0,0 +1,58 @@
+{ stdenv, lib, util-linux, fetchFromGitHub, bash, coreutils, ethtool, gnugrep, makeWrapper }:
+let
+  version = "20211028.00";
+in stdenv.mkDerivation {
+  inherit version;
+  pname = "guest-configs";
+
+  src = fetchFromGitHub {
+    owner = "GoogleCloudPlatform";
+    repo = "guest-configs";
+    rev = version;
+    sha256 = "sha256-0SRu6p/DsHNNI20mkXJitt/Ee5S2ooiy5hNmD+ndecM=";
+  };
+
+  buildInputs = [ bash makeWrapper ];
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    # allows to install the package in `services.udev.packages` in NixOS
+    mkdir -p $out/lib/udev $out/bin
+
+    cp -r "src/lib/udev/rules.d" $out/lib/udev
+    cp "src/lib/udev/google_nvme_id" $out/bin
+    cp "src/usr/bin/google_set_multiqueue" $out/bin
+
+
+    # sysctl snippets will be used by google-compute-config.nix
+    cp -r "src/etc/sysctl.d" $out
```

```suggestion
    cp -r src/etc/sysctl.d $out
```
Please don't quote literal paths.

---

?

---

Ignoring unless proven it could backfire.

---

Quoting literal paths does nothing, so it can also be removed. This is done nowhere else in the code and it prevents globs from functioning properly.